terms/neuro/diet.json

```diff
@@ -0,0 +1,33 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://repo-prod.prod.sagebase.org/repo/v1/schema/type/registered/sage.annotations-neuro.diet-0.0.1",
```

Comment:
Would we potentially want to call this something like "animalDiet," in case we need "diet" as a key for human data at some point? Possibly the gut microbiome data?

Reply:
animalDiet makes sense

Reply:
Since this is term being added for MODEL-AD, would it make sense to have a more specific name like:
- rodentDiet
- mouseDiet

Reply:
Since this will be specific to model-ad, I'll defer to you on which name would be most appropriate!

Reply:
Since it's model-ad specific I defer to your judgement on which specific term would be best!